pkg/app/master/commands/profile/handler.go

```diff
@@ -119,7 +119,7 @@ func OnCommand(
 	errutil.FailOn(err)
 
 	if gparams.Debug {
-		version.Print(fmt.Sprintf("cmd=%s", Name), logger, client, false, gparams.InContainer, gparams.IsDSImage)
+		version.Print(xc, fmt.Sprintf("cmd=%s", Name), logger, client, false, gparams.InContainer, gparams.IsDSImage)
```

Comment:
We need to refactor the `prefix` field to version.Print(), so it can be used with the xc call. We should repurpose the field (and possibly rename it) to specify the command type instead of passing ` fmt.Sprintf("cmd=%s", Name)`, so the refactored call would look like this: `version.Print(xc, Name, logger, client, false, gparams.InContainer, gparams.IsDSImage)` and then in the `version.Print` implementation we need to add an ovar param that we specify:
```
ov := ovars{
                "cmd":        cmdNameParam,
		"mode":        object.Mode,
```

I
version.Print(xc,, logger, client, false, gparams.InContainer, gparams.IsDSImage)

Reply:
Sure @kcq, working on it, however, couple of doubts here:
We want refactored version like this `version.Print(xc, Name, logger, client, false, gparams.InContainer, gparams.IsDSImage)` or like this `version.Print(xc,, logger, client, false, gparams.InContainer, gparams.IsDSImage)`? 

Also, will it be `"cmd":        cmdNameParam,` or `"cmd":        Name,` (the `Name` here refers to the one being passed, we can rename it to `cmdNameParam` if that is what you suggested?

Reply:
yes, it should be `version.Print(xc, Name, logger, client, false, gparams.InContainer, gparams.IsDSImage)`.

`Name` is what we'd pass to the `version.Print` call and the name for this parameter in the function definition itself can be anything (i used `cmdNameParam` as a potential name there, but you can pick a different name. using `Name` as the parameter variable name isn't  great because the function parameter names should start with a lower case letter).

Reply:
Got it, will do accordingly then! Thanks so much for making this clear!

Reply:
Tried to fix it, @kcq! Do review and let me know, thanks for the feedback!